devel/clavus/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo  1.0
+
+github.setup        peter-mbx clavus 0.0.1
+
+categories          devel
+revision            0
+
+maintainers         {gmail.com:pietro.mobrici} openmaintainer
+license             MIT
+
+description         Clavum Lateris
+long_description    In vetusta bibliotheca, clavum lateris aperiebat fores librarii occultum, ubi sapientia et arcana conservabantur.
+
+checksums           rmd160  d826b4c4370863a74950c9f923e7fa7e32971ec8 \
+                    sha256  9d74fea39c41056bcfab0320eeca3c3f61c34aed2d602b9b3e5894e462973fb2 \
+                    size    11434
+
+use_configure       no
+
+build.pre_args      {--release -v}
```

Comment:
These flags are already set by the `cargo` portgroup, so you don't need to set them again.

One thing worth mentioning is that `cargo` portgroup does build with `--frozen`, as MacPorts wants to track, checksum, and download all the crates itself, apart from `cargo`.  You would specify them using a `cargo.crates` section in your Portfile. Example here: https://github.com/macports/macports-ports/blob/master/sysutils/hyperfine/Portfile#L41

You can generate that using `cargo2port`.

If you'd prefer to just let `cargo` download and build as it normally does, you can add:

```
cargo.offline_cmd     {}
```